lib/ctbl.gi

```diff
@@ -4991,9 +4958,11 @@ BindGlobal( "CharacterTableDisplayDefault", function( tbl, options )
        elif centralizers = true then
           Print( "\n" );
           for i in [col..col+acol-1] do
-             fak:= Factors(Integers, tbl_centralizers[classes[i]] );
+             fak:= Factors( Integers, tbl_centralizers[ classes[i] ] );
```

Comment:
Optional: This could also be changed to `fak := PrimeDivisors( tbl_centralizers[ classes[i] ] );`, then the call to `Set` in the next line and the check for `prime <> 1` later would not be needed.